policyengine_us/variables/gov/states/ct/tax/income/credits/ct_eitc.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class ct_eitc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Connecticut Earned Income Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://portal.ct.gov/-/media/DRS/Forms/2022/Income/Schedule-CT-EITC_1222.pdf"
+    defined_for = StateCode.CT
+
+    def formula(tax_unit, period, parameters):
```

Comment:
this looks great, now we need to add the condition from part (c) of the law- where a separate state return is filed. An example is the iowa (ia) model - its itemized deductions and the ia_prorate_fraction.py variable

Reply:
To include this condition, would I adjust the rate.yaml and ct_eitc.py files?

Reply:
The parameter file should be good, need to adjust the .py file

Reply:
I completed part of it, but I'm unsure how to distinguish the AGI values for when someone files both a separate and joint return, so then that fraction (separate AGI / joint AGI) can be multiplied by the other values.